### PR TITLE
Update Halo MCC to add notes about functionality

### PR DESCRIPTION
**Pull request by udekmp**
I added notes that specify that:
"Singleplayer, co-op, and custom matches work"

### CODE REVIEW

File: games.json [hunk 1]
```diff
@@ -9,7 +9,9 @@
         "anticheats": [
             "Easy Anti-Cheat"
         ],
-        "notes": [],
+        "notes": [
+        Singleplayer, co-op, and custom matches work
```
Comment (Curve) [inline games.json:13]: Invalid formatting

**Review by Curve — CHANGES_REQUESTED** [review]

**Comment by udekmp** [comment]
I may have still formatted incorrectly... but I attempted to fix it.

**Comment by Curve** [comment]
Yep still wrong, I'll change it manually

**Comment by udekmp** [comment]
> Yep still wrong, I'll change it manually

Thank you!

**Pull request merged**